Src/GBX.NET/Engines/Game/CGameCtnCollection.chunkl

```diff
@@ -67,28 +67,28 @@ CGameCtnCollection 0x03033000
  ident Vehicle
 
 0x00B
- int
+ int BlocksShadow
 
 0x00C
  bool IsEditable
- int
+ int<ECollectionType> CollectionType
 
-0x00D
+0x00D //IconFid, IconSmallFid
 
 0x00E
  int SortIndex
 
 0x011
- int
+ int DefaultDecoration
```

Comment:
looking at the chunkl file, there's another property named like this in `0x008`, so, with certain expectation, here it should be
```
CGameCtnDecoration DefaultDecoration (external)
```
mb, didn't notice the further issue here